autosubmit/job/job_list.py

```diff
@@ -3253,12 +3237,9 @@ def retrieve_times(status_code, name, tmp_path, make_exception=False, job_times=
         energy = 0
         seconds_queued = 0
         seconds_running = 0
-        queue_time = running_time = 0
         submit_time = datetime.timedelta()
         start_time = datetime.timedelta()
         finish_time = datetime.timedelta()
-        running_for_min = datetime.timedelta()
-        queuing_for_min = datetime.timedelta()
```

Comment:
I think these values have already been deleted elsewhere. Can you rebase it, please, @VindeeR ?

Reply:
Ah, I think I saw this code being changed somewhere else and got confused. Sorry.